lib/MySQL_Thread.cpp

```diff
@@ -4,6 +4,8 @@
 #include "cpp.h"
 #include "MySQL_Thread.h"
 
+#define PROXYSQL_LISTEN_LEN 1024
```

Comment:
I would add a comment to explain why I needed such a large connection backlog. Mabe make a follow-up issue to make it configurable.